src/libs/actions/Policy.js

```diff
@@ -213,6 +213,69 @@ function hasActiveFreePolicy(policies) {
     return true;
 }
 
+/**
+ * Build optimistic data for adding users from announce room
+ * @param {String} policyID
+ * @param {Object} invitedEmailsToAccountIDs
+ * @returns {Object}
+ */
+function createOptimisticAnnounceRoomMembers(policyID, invitedEmailsToAccountIDs) {
```

Comment:
NAB: Maybe we can call it `buildAnnounceRoomMembersOnyxData`? It's more than just optimistic data.

Reply:
NAB: It looks like we also only need the invited accountIDs here rather than invitedEmailsToAccountIDs